includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1919,10 +1920,23 @@ protected function check_response_for_errors( $response ) {
 					$response_code
 				);
 			} elseif ( isset( $response_body['error'] ) ) {
+				$response_body_error_code = $response_body['error']['code'] ?? null;
+				$payment_intent_status    = $response_body['error']['payment_intent']['status'] ?? null;
+
+				// Check for an 'amount_too_large' error code and a 'requires_capture' payment intent status.
+				// We redact the API error message to prevent prompting the merchant to contact Stripe support
+				// when attempting to manually capture an amount greater than what's authorized. Contacting support is unnecessary in this scenario.
+				if ( 'amount_too_large' === $response_body_error_code && 'requires_capture' === $payment_intent_status ) {
```

Comment:
We a constant for this intent status in `\Intent_Status`

Reply:
Using the constant here https://github.com/Automattic/woocommerce-payments/pull/7581/commits/f8ad535d33cf07ff1856a682bea48145c2c45e19 thanks!